src/pages/TestDrive/TestDrive.tsx

```diff
@@ -0,0 +1,117 @@
+import { Button, Container, Form, Card } from "react-bootstrap";
+import { useLocation, useParams, Link, useNavigate } from "react-router-dom";
+import React, { useState } from "react";
+import "./TestDrive.scss";
+function TestDrive() {
+  let navigate = useNavigate();
+  const [mobileValid, setMobileValid] = useState(false);
+  const [emailValid,setEmailValid] = useState(false);
+  var pattern = /^(\+\d{1,3}[- ]?)?\d{10}$/;
+  let emailPattern = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/
+  const [validated, setValidated] = useState(false);
+
+  const handleSubmit = (event: any) => {
+    const form = event.currentTarget;
+    event.preventDefault();
+    event.stopPropagation();
```

Comment:
```suggestion
    event.preventDefault();
```
just prevent Default should suffice